### PR TITLE
task/3698 my orders page redesigned

### PR DESCRIPTION
**Pull request by tiwariav**

### CODE REVIEW

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -474,6 +461,11 @@ msgstr "Du kannst dich nun"
 msgid "Sorry. Your request is invalid."
 msgstr "Entschuldigung, deine Anfrage ist ungültig."
 
+#, fuzzy
+#| msgid "Credit Card"
+msgid "Invalid credit card"
+msgstr "Invalid Kreditkarte"
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Ungültige Kreditkarte"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -441,12 +434,6 @@ msgstr ""
...
 msgstr "NEUE VM"
 
-msgid "Page"
-msgstr ""
-
-msgid "of"
-msgstr ""
-
 msgid "login"
 msgstr "einloggen"
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: "Einloggen"

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 3]
```diff
@@ -474,6 +461,11 @@ msgstr "Du kannst dich nun"
 msgid "Sorry. Your request is invalid."
 msgstr "Entschuldigung, deine Anfrage ist ungültig."
 
+#, fuzzy
```
Comment (pcoder) [inline hosting/locale/de/LC_MESSAGES/django.po]: I think, we better remove fuzzy here

File: hosting/templates/hosting/orders.html [hunk 4]
```diff
@@ -3,95 +3,65 @@
...
+                <th>{% trans "Order Nr." %}</th>
+                <th>{% trans "Date" %}</th>
+                <th>{% trans "Amount" %}</th>
+                <th>{% trans "Status" %}</th>
+                <th></th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td class="xs-td-inline" data-header="{% trans 'Order Nr.' %}">{{ order.id }}</td>
```
Comment (pcoder) [inline hosting/templates/hosting/orders.html]: Is Nr the shorthand for number?

Comment (tiwariav) [inline hosting/templates/hosting/orders.html]: Yes, @verysanghee is it how its supposed to be?

Comment (tiwariav) [inline hosting/templates/hosting/orders.html]: Yes it is.

Comment (pcoder) [inline hosting/templates/hosting/orders.html]: Nr indeed seems to be the shorthand for number in German https://en.wikipedia.org/wiki/Numero_sign
I would never use it in English, just saying ..

**Pull request merged**